From 9e7537caddb9c6adc720bb3322a7512cf51ab025 Mon Sep 17 00:00:00 2001
From: Richard Russon <rich@flatcap.org>
Date: Fri, 17 Apr 2020 15:36:58 +0100
Subject: [PATCH] ensure OP_NULL is always first

A recent refactoring altered the ordering of the OPs when autocrypt was
enabled.  This mean that OP_NULL wasn't 0 any more.

Fixes: #2268
---
 opcodes.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opcodes.h b/opcodes.h
index 701df71f86..3984d3a038 100644
--- a/opcodes.h
+++ b/opcodes.h
@@ -37,7 +37,6 @@
 #endif
 
 #define OPS_CORE(_fmt) \
-  _fmt(OP_NULL,                           N_("null operation")) \
   _fmt(OP_ATTACH_COLLAPSE,                N_("toggle display of subparts")) \
   _fmt(OP_ATTACH_VIEW_MAILCAP,            N_("force viewing of attachment using mailcap")) \
   _fmt(OP_ATTACH_VIEW_TEXT,               N_("view attachment as text")) \
@@ -317,6 +316,7 @@
   _fmt(OP_COMPOSE_SMIME_MENU,             N_("show S/MIME options")) \
 
 #define OPS(_fmt) \
+  _fmt(OP_NULL,                           N_("null operation")) \
   OPS_AUTOCRYPT(_fmt) \
   OPS_CORE(_fmt) \
   OPS_SIDEBAR(_fmt) \
